_pages/home/updating-to-version-5/updating-to-version-5.md

```diff
@@ -70,10 +79,94 @@ With the release of Spartacus 5.0, the hosting service of SAP Commerce Cloud can
 
    For more information, see [Adding Applications for JavaScript Storefronts](https://help.sap.com/docs/SAP_COMMERCE_CLOUD_PUBLIC_CLOUD/b2f400d4c0414461a4bb7e115dccd779/63577f67a67347bf9f4765a5385ead33.html).
 
+## Removal of ng-bootstrap
+
+ng-bootstrap has been removed from Spartacus 5.0 and is no longer a dependency. You can continue to use ng-bootstrap in your Spartacus application, but this may require upgrading additional third party dependencies. For more information, see the list of [ng-bootstrap dependencies](https://www.npmjs.com/package/@ng-bootstrap/ng-bootstrap).
+
+If you no longer need ng-bootstrap in your Spartacus application, you can run the command `yarn remove @ng-bootstrap/ng-bootstrap`.
+
+## Removal of ModalService and Related Code
+
+The `ModalService`, `ModalDirective`, `ModalDirectiveOptions`, `ModalDirectiveService`, `ModalOptions` and `ModalRef` have all been removed from Spartacus 5.0. If you are using the `ModalService` and related code in your Spartacus application, you can replace it with the `LaunchDialogService` provided by Spartacus.
+
+To open a modal, use the `openDialog` or `openDialogAndSubscribe` methods. The following is an example:
+
+```ts
+const dialog = this.launchDialogService.openDialog(
+    LAUNCH_CALLER.CLOSE_ACCOUNT,
+    this.element,
+    this.vcr
+  );
+
+dialog?.pipe(take(1)).subscribe();
+```
+
+The `LaunchDialogService` uses `LAUNCH_CALLER` to open a modal, so you need to augment `LAUNCH_CALLER` with a specific key for the component that you want to open in the modal. The following is an example:
+
+```ts
+import '@spartacus/storefront';
+
+declare module '@spartacus/storefront' {
+    const enum LAUNCH_CALLER {
+       CLOSE_ACCOUNT = 'CLOSE_ACCOUNT',
+    }
+}
+```
+
+Once you have a key for your component, you need to provide a configuration to the component module that is specified for your modal. The following is an example:
+
+```ts
+import { DIALOG_TYPE, LayoutConfig } from '@spartacus/storefront';
+import { CloseAccountModalComponent } from './close-account-modal.component';
+
+export const defaultCloseDialogModalLayoutConfig: LayoutConfig = {
+    launch: {
+       CLOSE_ACCOUNT: {
+          inline: true,
```

Comment:
Should we also mention about other rendering strategies like `inlineRoot` for example?